### ISSUE TITLE
Console option F8 no longer shown in Webinterface Config

### ISSUE THREAD

**Issue by tin-nl** [atarijookie/ce-atari#115]
@atarijookie: any reason the user can't use the console in webinterface? F8 was explicitly removed. AFAIR it worked before. I'd like to re-add F8 if there's no reason not to.

**Comment by atarijookie** [comment]
There is no real technical reason for not adding console access through web
interface, only security issue - web interface and that config console
through web interface are not protected by any password, but as this is
mostly run by users on local networks, even this shouldn't be a big
concern...

The only reason I removed that F8 text from web config console is that it
didn't work / do anything last time I tried, so I removed the text to not
provoke users :) if it works, text can be returned back. If it doesn't
work, some fixing maybe would be needed if you like to have this feature
through web...
On 14 Dec 2016 15:26, "tin-nl" <notifications@github.com> wrote:

@atarijookie <https://github.com/atarijookie>: any reason the user can't
use the console in webinterface? F8 was explicitly removed. AFAIR it worked
before. I'd like to re-add F8 if there's no reason not to.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/atarijookie/ce-atari/issues/115>, or mute the thread
